### PR TITLE
initial feedback styling and update to var in summary list causing co…

### PR DESCRIPTION
**Pull request by captain-pj**
…nflicts

Want some initial feedback on what I have been working on with the feedback and notifications.

### CODE REVIEW

**Comment by captain-pj** [comment]
Updated the general design. Since I was going off the original screengrab in the ticket, I wasn't sure if we needed all of the color possibilities or not. Let me know if it is still too colorful or any other feedback you might have.

**Comment by williamdingwall** [comment]
**Re-wrote a lot of this and updated the progress component**

File: src/css/components/Feedback.scss [hunk 1]
```diff
@@ -0,0 +1,120 @@
+@import "../config/index.scss";
```
Comment (saralohr) [inline src/css/components/Feedback.scss]: Do we need to do this still? I think we were doing this for the sake of CSS modules in LFE.

File: src/css/components/ProgressBar.scss [hunk 2]
```diff
@@ -1,135 +1,301 @@
...
+    &, .ProgressBar-status {
+      border-radius: scaleGrid(8) / 2;
     }
+  }
+
+}
+
+// * 3. Precise Modifier
+
+.ProgressBar.ProgressBar--precise {
```
Comment (saralohr) [inline src/css/components/ProgressBar.scss]: Can this just be `.ProgressBar--precise`?

Comment (williamdingwall) [reply src/css/components/ProgressBar.scss]: needed it for specificity

File: src/css/components/Feedback.scss (hunk 1 above)
Comment (williamdingwall) [reply src/css/components/Feedback.scss]: yeah we could probably drop it - just keeping this file consistent with the rest

File: src/css/components/ProgressBar.scss (hunk 2 above)
Comment (williamdingwall) [reply src/css/components/ProgressBar.scss]: ..actually nevermind, updated it. Was running into a specificty issue but not an issue. also moved this to a new PR : https://github.com/teamsnap/teamsnap-ui/pull/74

**Pull request closed without merging**